src/controllers/walletControllers.js

```diff
@@ -28,8 +28,45 @@ const createWallet = async (req, res, next) => {
   } catch (error) {
     next(error);
   }
+}
+
+// Fund Users wallet
+const fundWallet = async (req, res, next) => {
+  try {
+    // Get amount user wants to fund wallet with and the transaction description from the user
+    const { accountBank, accountNumber, fundNarration, amountFund } = req.body;
+    const fundCurrency = 'NGN';
+    const fundReference = `transfer-${Date.now()}`;
```

Comment:
Date now is not a robust transaction reference. What happens if multiple transactions happen at the same time. Better to use a uuid then you can attached a date now to it if you want, but I would advise that you did not